symengine/tests/basic/test_functions.cpp

```diff
@@ -649,6 +649,7 @@ TEST_CASE("Csc: functions", "[functions]")
     REQUIRE(eq(*r1, *r2));
 
     CHECK_THROWS_AS(csc(mul(integer(7), pi)), std::runtime_error);
+    CHECK_THROWS_AS(csc(mul(integer(0), integer(0))), std::runtime_error);
```

Comment:
This works fine when using `integer(0)`. But the other test doesn't. Sorry, I assumed that similar error will show up in this test too.